dataplane-webserver/src/backups/mod.rs

```diff
@@ -280,108 +416,96 @@ async fn run_backup_on_instance_pod(
         "Found primary database pod for backup"
     );
 
-    // Check if temback exists and install if needed
-    let temback_path = format!("{}/temback", TEMBACK_INSTALL_DIR);
-    let check_cmd = vec!["test", "-f", temback_path.as_str()];
+    // Install temback if needed
+    if let Err(e) = install_temback(&pods_api, pod_name, config).await {
+        return Ok(BackupResult::Failed(e.to_string()));
+    }
+
+    // Execute temback command in the pod
     let attach_params = AttachParams::default()
         .container("postgres")
         .stderr(true)
         .stdout(true)
         .stdin(false);
 
-    let check_result = pods_api.exec(pod_name, check_cmd, &attach_params).await;
-    if check_result.is_err() {
-        tracing::info!(
-            pod = %pod_name,
-            version = %config.temback_version,
-            "temback not found, installing..."
-        );
-
-        // Install temback using version from config
-        let install_cmd_str = TEMBACK_INSTALL_CMD_TEMPLATE
-            .replace("{version}", &config.temback_version)
-            .replace("{install_dir}", TEMBACK_INSTALL_DIR);
-        let install_cmd = vec!["sh", "-c", &install_cmd_str];
-
-        let mut install_output = pods_api
-            .exec(pod_name, install_cmd, &attach_params)
-            .await
-            .map_err(|e| ErrorInternalServerError(format!("Failed to install temback: {}", e)))?;
-
-        // Check for installation errors
-        if let Some(mut stderr) = install_output.stderr() {
-            let mut error_msg = String::new();
-            stderr.read_to_string(&mut error_msg).await.map_err(|e| {
-                ErrorInternalServerError(format!("Failed to read install error output: {}", e))
-            })?;
-
-            if !error_msg.is_empty() {
-                return Ok(BackupResult::Failed(format!(
-                    "Failed to install temback: {}",
-                    error_msg
-                )));
-            }
-        }
-
-        tracing::info!(
-            pod = %pod_name,
-            "Successfully installed temback"
-        );
-    }
-
-    // Execute temback command in the pod
     let backup_cmd = vec![
         "/var/lib/postgresql/data/temback",
         "--name",
         namespace,
         "--compress",
         "--clean",
+        "--cd",
+        "/tmp",
         "--bucket",
         &bucket_name,
         "--dir",
         &bucket_path,
+        "--pass",
+        &password,
+        "--host",
+        "localhost",
+        "--user",
+        "postgres",
```

Comment:
You can omit these now if it they're not actually needed to connect. Presumably if it's running inside the pod where localhost connections are trusted, you don't need any of this stuff.

Also, do we want to use `--text`?

Reply:
Ok just confirmed that removing the host, user and pass work.  I don't know why we would use `--text` for this situation